acquisitions/src/main/resources/thunderjet/mod-orders/features/open-order-failure-with-expenditure-restrictions.feature

```diff
@@ -0,0 +1,102 @@
+@parallel=false
+# Created for MODORDERS-1049
+# Related to MODORDERS-528 and open-order-failure-side-effects.feature
+Feature: Open order failure with expenditure restrictions
+
+  Background:
+    * print karate.info.scenarioName
+    * url baseUrl
+    * callonce loginAdmin testAdmin
+    * def okapitokenAdmin = okapitoken
+    * callonce loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json' }
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*' }
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
+    * def createOrderLine = read('classpath:thunderjet/mod-orders/reusable/create-order-line.feature')
+
+    * def ledgerId = callonce uuid1
+    * def fundId = callonce uuid2
+    * def budgetId = callonce uuid3
+    * def orderId = callonce uuid4
+    * def poLineId = callonce uuid5
+    * def titleOrPackage = callonce uuid6
+
+
+  Scenario: Create finances
+    * configure headers = headersAdmin
+    * def v = call createLedger { id: "#(ledgerId)", fiscalYearId: "#(globalFiscalYearId)", restrictEncumbrance: false, restrictExpenditures: true }
+    * def v = call createFund { id: "#(fundId)", ledgerId: "#(ledgerId)" }
+    * def v = call createBudget { id: "#(budgetId)", fundId: "#(fundId)", allocated: 5 }
+
+
+  Scenario: Create an order and line
+    * def v = call createOrder { id: "#(orderId)" }
+    * def v = call createOrderLine { id: "#(poLineId)", orderId: "#(orderId)", fundId: "#(fundId)", listUnitPrice: 10, titleOrPackage: titleOrPackage }
```

Comment:
better to add comment that we are creating order line with bigger price that is allocated in budget to quicker understand why openning order failed

Reply:
OK